packages/espresso/android/build.gradle

```diff
@@ -79,8 +79,18 @@ dependencies {
 
     // Assertions
     api 'androidx.test.ext:junit:1.1.5'
-    api 'androidx.test.ext:truth:1.5.0'
-    api 'com.google.truth:truth:1.1.3'
+    // Don't make these 'api', otherwise another workaround for guava 32.1.+ on gradle 6.x is needed:
```

Comment:
They might (I don't know whether it was intended)

It just lesser evil between workarounds I've found:
- either demote api here to implemenation (and since all_packages are built with it - it seemed ok to me)
- or add api to guava dependency (see https://github.com/flutter/packages/pull/6535#discussion_r1567278787 and https://github.com/flutter/packages/pull/6535#discussion_r1567520344)
- bumg minimal supported gradle to 7.x, since gradle 6 only is affected

Reply:
> and since all_packages are built with it - it seemed ok to me

The deciding factor isn't whether one particular configuration happened to build, it's whether types from these dependencies are part of the public API surface.

Reply:
Yes, and they are part of public API.
I just don't know what workaround to pick from, or any other solution
What do you prefer from mentioned above?

Reply:
@stuartmorgan if we can't decide now on concrete workaround, I'd propose
- creating separate issue on guava 32.1.+ with gradle 6. It happened that media3 1.3 depends on newer guava, but there are other plugins - camera, espresso, google_sign_in, etc - which face the same issue once they bump guava dependency
- leave media3 at 1.1.1 or 1.2.1, remove commits a72c25811232b5264ad3a90e9f1f54d695b330c3 20d6823fa19fc90fdb5c20747d6c37d349cd6034

what do you think?

Reply:
I'll remove workarounds for gradle 6 since all_projects uses 7 now - https://github.com/flutter/packages/pull/6591